opensrp-chw/src/test/java/org/smartregister/chw/util/UtilsTest.java

```diff
@@ -123,4 +144,22 @@ public void testAddHyphenBetweenNumbers() {
         Assert.assertEquals("Ali is around 2-3 years old", addHyphenBetweenNumbers("Ali is around 2 3 years old"));
         Assert.assertEquals("Ali is around 2 years old", addHyphenBetweenNumbers("Ali is around 2 years old"));
     }
+
+    @Test
+    public void testSetupBottomNavigation() {
+
+        when(mockBottomNavigationView.getMenu()).thenReturn(menu);
+
+        org.smartregister.chw.util.Utils.setupBottomNavigation(mockBottomNavigationHelper, mockBottomNavigationView, mockListener);
+        verify(mockBottomNavigationView).setLabelVisibilityMode(LabelVisibilityMode.LABEL_VISIBILITY_LABELED);
+        verify(mockBottomNavigationView).inflateMenu(R.menu.bottom_nav_menu);
+        verify(mockBottomNavigationHelper).disableShiftMode(mockBottomNavigationView);
+        verify(mockBottomNavigationView).setOnNavigationItemSelectedListener(mockListener);
```

Comment:
Can we verify if the QR, job aids and reports items are removed if the flavor does not have them